content/applications/hr/time_off.rst

```diff
@@ -221,34 +221,30 @@ Plans`. Then, click the :guilabel:`New` button, which reveals a blank accrual pl
+- :guilabel:`Name`: Enter the accrual plan name.
+- :guilabel:`Accrued Gain Time`: Select when the employee begins to accrue time off, either
   :guilabel:`At the start of the accrual period` or :guilabel:`At the end of the accrual period`.
-- :guilabel:`Carry-Over Time`: select when the employee received previously earned time. The options
+- :guilabel:`Carry-Over Time`: Select when the employee received previously earned time. The options
   are:
 
-  - :guilabel:`At the start of the year`: select this if the accrual rolls over on January 1 of the
+  - :guilabel:`At the start of the year`: Select this if the accrual rolls over on January 1 of the
     upcoming year.
-  - :guilabel:`At the allocation date`: select this if the accrual rolls over as soon as time is
+  - :guilabel:`At the allocation date`: Select this if the accrual rolls over as soon as time is
     allocated to the employee.
-  - :guilabel:`Other`: select this option if neither of the other two options are applicable. When
+  - :guilabel:`Other`: Select this option if neither of the other two options are applicable. When
     selected, a :guilabel:`Carry-Over Date` field appears. Select the date using the two drop-down
     menus, one for the day and one for the month.
 
-- :guilabel:`Based on worked time`: enable this option if time off accrual is determined by the
+- :guilabel:`Based on worked time`: Enable this option if time off accrual is determined by the
   employee's worked hours. Days **not** considered as worked time do **not** contribute to the
   accrual plan in Odoo.
-- :guilabel:`Milestone Transition`: this selection determines when employees move up to a new
-  milestone. If they qualify to change milestones in the middle of a pay period, select if the
-  employee changes milestones :guilabel:`Immediately` or :guilabel:`After this accrual's period`
-  (after the current pay period).
-
-  .. important::
-     The :guilabel:`Milestone Transition` field is **only** visible after a minimum of two
-     :ref:`rules <time_off/rules>` have been configured on the accrual plan.
-
-- :guilabel:`Company`: using the drop-down menu, select the company the accrual plan applies to. If
-  left blank, the accrual plan can be used for all companies. This field only appears in a
+- :guilabel:`Milestone Transition`: This field is **only** visible after a minimum of two
+  :ref:`rules <time_off/rules>` have been configured on the accrual plan. This selection determines
+  when employees move up to a new milestone. If they qualify to change milestones in the middle of a
+  pay period, decide whether the employee changes milestones :guilabel:`Immediately` or
+  :guilabel:`After this accrual's period` (after the current pay period).
+- :guilabel:`Company`: Using the drop-down menu, select the company the accrual plan applies to. If
+  left blank, the accrual plan can be used for all companies. This field **only** appears in a
```

Comment:
Qualifying context like "this field only appears in a multi-company database" should be read first before the field description in order to save the reader time on what's relevant for them in the doc.